### ISSUE TITLE
[Bug] Animeworld.tv: source broken

### ISSUE THREAD

**Issue by Palloxin** [jmir1/aniyomi-extensions#444]
## Device information
- Tachiyomi version: 0.12.3.6
- Android version: 8

## Source information
- Name: animeworld.tv (italian)
- Extension version: 12.5

## Steps to reproduce
1. open an anime
2. try to watch an episode

### Expected behavior
the episode should start

### Actual behavior
![immagine](https://user-images.githubusercontent.com/75091899/160853817-524363b8-d499-4a02-9e56-ce0713f0ebf0.png)

**Comment by ahmedgamal17** [comment]
Try vpn

**Comment by Palloxin** [comment]
> Try vpn

tried the vpn, doesn't work.
on the browser the site works fine (without vpn)

**Comment by ahmedgamal17** [comment]
Check the dood download link in the episode page if it's working or not

**Comment by Palloxin** [comment]
> Check the dood download link in the episode page if it's working or not

no, doesn't work. dood link is the download button of the episode on the site i think (just checked with inspect element)

**Comment by Palloxin** [comment]
https://www.animeworld.tv/play/sabikui-bisco.qUN6L/mpqxpT
![immagine](https://user-images.githubusercontent.com/75091899/160860942-083553db-686f-4a36-a1f7-20b0aedfb6fe.png)

**Comment by Palloxin** [comment]
the site dood works again, so the extension is no longer broken

**Comment by Palloxin** [comment]
The site has up and down moments, it is better if you change the episode source pls, i will reopen
